Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9.0-alpine
+FROM python:3.11-slim-bullseye
```

Comment:
Debian images are less trouble and faster than Alpine images, and upgraded from Python 3.9 to Python 3.11.